### DISCUSSION TITLE
Connectors Step Functions destination permissions are missing the APIs to read and update Distributed Map runs.

### DISCUSSION THREAD

**Discussion by wagneraw** [aws/serverless-application-model#2956] [Ideas]
With the release of Step Functions Distributed Map, there are now three new API actions. Two read API [ListMapRuns](https://docs.aws.amazon.com/step-functions/latest/apireference/API_ListMapRuns.html), [DescribeMapRun](https://docs.aws.amazon.com/step-functions/latest/apireference/API_DescribeMapRun.html) and a write API [UpdateMapRun](https://docs.aws.amazon.com/step-functions/latest/apireference/API_UpdateMapRun.html). These should be added to the the read and write permissions for Connectors with a destination of step functions state machine.

**Comment by drew-clements** [comment]
@wagneraw do you know what the eta is for adding Step Functions Distributed Map into SAM CLI?
